openedx/frontend-app-payment
# REV-2652: bump versions for frontend-platform and axios

The Axios package had a security vulnerability, fixed in version 0.26.0. This PR: 
- bumps axios version to 0.26.0
- bumps frontend-platform to 1.15.2 (that's version that uses the new axios)

Note: while this branch removes the vulnerable advisories coming from axios and frontend-platform, we do still have 4 coming from frontend-build, making the node_js CI job fail (Vulnerable advisories are: 1041054, 1041107, 1041286, 1041350). This is because we the update to have frontend-app-payment use the newer frontend-build (REV-2912) wasn't able to deploy, this issue is in investigation here: 
https://github.com/openedx/frontend-wg/issues/85


## Supporting information
(https://openedx.atlassian.net/browse/REV-2652)


## Testing instructions
- pull down this branch
- run 'npm test' to see that everything passes
- local checkout page loads (apart from known issue with flex form fields: [REV-2561](https://openedx.atlassian.net/browse/REV-2561))


## Checklist
- [ ] Consider PCI compliance impact and whether this PR changes how credit card information is handled.
- [ ] Intend to [release and monitor](https://openedx.atlassian.net/wiki/spaces/RS/pages/1835106870/How+to+contribute+to+our+repositories) this PR promptly after merge.
